recipes/arccommander/build.sh

```diff
@@ -0,0 +1,10 @@
+cd $SRC_DIR
+dotnet tool restore
+dotnet publish src/ArcCommander -c Release -p:UseAppHost=false
+PREFIX=$(echo "${PREFIX}" | tr '\\' '/')
+DOTNET_ROOT="${PREFIX}/lib/dotnet"
+TOOL_ROOT=$DOTNET_ROOT/tools/arccommander
```

Comment:
```suggestion
TOOL_ROOT="${DOTNET_TOOLS}/arccommander"
```
https://github.com/conda-forge/dotnet-feedstock/blob/b10afdcf03e444fa5e8f89bf389be83bed3bb1af/recipe/activate.d/dotnet-root.sh#L18